tests/signature_aggregator_api.go

```diff
@@ -193,7 +203,9 @@ func sampleMetrics(port uint16) map[string]uint64 {
 				log.Debug("Found metric line", "line", line)
 				parts := strings.Fields(line)
 
-				// Fetch the metric count from the last field of the line
+				metricName = strings.Replace(parts[0], "U__signature_2d_aggregator_", "", 1)
```

Comment:
JW how does this work with multiple subnets? Can we add a test case for that?

Reply:
note how I constructed the metric name using the subnet ID here: https://github.com/ava-labs/awm-relayer/pull/434/files#diff-973f1f8ae5b8f6acd388e8c2fdaa6e08f999b4211536e8e3258b5df24de79d04R135

in short, the metric name/key for the existing test case comes out as `connected_stake_weight_percentage{subnetID="11111111111111111111111111111111LpoYY"}`.

If I understand correctly, we can't send a message from any subnets until the ACP-118 stuff is merged. (Right?) So at this time I can't add a test case for a different subnet, but I think the existing test does a good job of showing how it works.